src/lib/components/ui/ScrollView/ScrollView.jsx

```diff
@@ -75,6 +101,47 @@ export const ScrollView = (props) => {
     debounce,
   );
 
+  const autoScrollChildrenKeys = autoScroll !== 'off' && children && React.Children
+    .map(children, (child) => child.key)
+    .reduce((reducedKeys, childKey) => reducedKeys + childKey, '');
+  const autoScrollChildrenLength = autoScroll !== 'off' && children && children.length;
+
+  /**
+   * If autoScroll is enabled, it automatically scrolls viewport element to the end of the
+   * the viewport when content is changed. It is performed only when viewport element is
+   * scrolled to the end of the viewport or when viewport element is in any position but
+   * autoScroll triggered by previous change is still in progress.
+   */
+  useLayoutEffect(() => {
+    if (autoScroll === 'off') {
+      return () => {};
+    }
+
+    const scrollViewContentElement = scrollViewContentEl.current;
+    const scrollViewViewportElement = scrollViewViewportEl.current;
+
+    const differenceX = direction === 'horizontal' ? scrollViewContentElement.offsetWidth : 0;
+    const differenceY = direction !== 'horizontal' ? scrollViewContentElement.offsetHeight : 0;
+
+    if (autoScroll === 'always') {
```

Comment:
I think for the `always` value an update should be also triggered after the viewport has been resized.